src/cpr_sdk/models/search.py

```diff
@@ -256,6 +256,15 @@ def validate(self):
             )
         return self
 
+    @model_validator(mode="after")
+    def concept_filters_not_set_if_documents_only(self) -> "SearchParameters":
+        """Ensure concept_filters are not set if browse mode (documents_only) is set."""
+        if self.concept_filters is not None and self.documents_only is True:
+            raise ValueError(
+                "Cannot set concept_filters when browse_mode is set. This is as concept_filters are only applicable to passages."
```

Comment:
"browse" is the backends term used to describe the documents_only, functionality and isnt used in the sdk, I think here it would be more appropriate to just say something more like "when only searching documents"

Reply:
Yes this makes absolute sense! Cheers